manifests/vsphere-7.0/vanilla/deploy/vsphere-csi-node-ds.yaml

```diff
@@ -0,0 +1,127 @@
+# Minimum Kubernetes version - 1.16
+# For prior releases make sure to add required --feature-gates flags
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: vsphere-csi-node
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: vsphere-csi-node
+  updateStrategy:
+    type: "RollingUpdate"
+  template:
+    metadata:
+      labels:
+        app: vsphere-csi-node
+        role: vsphere-csi
+    spec:
+      dnsPolicy: "Default"
+      containers:
+      - name: node-driver-registrar
+        image: quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
+        lifecycle:
+          preStop:
+            exec:
+              command: ["/bin/sh", "-c", "rm -rf /registration/csi.vsphere.vmware.com /var/lib/kubelet/plugins_registry/csi.vsphere.vmware.com /var/lib/kubelet/plugins_registry/csi.vsphere.vmware.com-reg.sock"]
+        args:
+        - "--v=5"
+        - "--csi-address=$(ADDRESS)"
+        - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
+        env:
+        - name: ADDRESS
+          value: /csi/csi.sock
+        - name: DRIVER_REG_SOCK_PATH
+          value: /var/lib/kubelet/plugins_registry/csi.vsphere.vmware.com/csi.sock
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: plugin-dir
+          mountPath: /csi
+        - name: registration-dir
+          mountPath: /registration
+      - name: vsphere-csi-node
+        image: vmware/vsphere-csi:<vsphere_csi_ver>
```

Comment:
What is the plan with `<vsphere_csi_ver>`? This should be addressed in upcoming PR, cannot be used otherwise.

Reply:
@alex1989hu driver images in the YAML will be replaced with following when release `v.2.0.0` is cut. 

```
gcr.io/cloud-provider-vsphere/csi/release/driver:v2.0.0
gcr.io/cloud-provider-vsphere/csi/release/syncer:v2.0.0
```

Reply:
when is the release?

Reply:
I also hope to learn that please. I urgently need a working image.

Reply:
working images, but not final release:
```
gcr.io/cloud-provider-vsphere/csi/ci/driver:v1.0.2-265-gbaaccdf
gcr.io/cloud-provider-vsphere/csi/ci/syncer:v1.0.2-265-gbaaccdf
```

Reply:
I also update this information in my own issue: #155 . I tested the latest image and it worked on kubernetes 1.16.6 with vSphere 7.0